components/shared/net/mime_classifier.rs

```diff
@@ -3,7 +3,8 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 use mime::{self, Mime};
-use net_traits::LoadContext;
+
+use crate::LoadContext;
 
 pub struct MimeClassifier {
```

Comment:
I moved this into `net_traits` since `script` does not depend on `net`. Since it is a relatively clean class with barely any dependencies, I suppose that's fine?